_posts/2024-12-19-cosmicqc.md

```diff
@@ -0,0 +1,313 @@
+  include figure.html
+  image="images/blog/cosmicqc/segmentation_cartoon.png"
+  width="100%"
+%}
+> *Simplified representation of single-cell segmentation.*
+
+In the Way Lab, we traditionally process single-cell data, so we expect heterogeneity within our samples.
+We do not want the heterogeneity influenced by technical factors, like poor segmentations.
+So, we want to remove any of these poor segmentation before we perform important pre-processing steps like normalization and feature selection.
+If we don't filter poor segmentations, our data processing will include errors, which will reduce the signal-to-noise ratio.
+
+Therefore, we perform quality control for single-cell morphology profiles after CytoTable but before pycytominer.
+But, what methodology exists so that we can perform this task?
+
+---
+
+## What quality control methods currently exist?
+
+If you are familiar with the RNA-seq world, you will know that the concept of *Single-cell quality control* itself is not new.
+It is a well known process within scRNA-seq pipelines where individual cells are filtered out using a metric(s) that detects if it is likely an artifact or contains noise.
+It is such a well-established part of this field that there even is [a whole website detected to scRNA tools](https://www.scrna-tools.org/), where you can filter for specifically [quality control tools](https://www.scrna-tools.org/tools?sort=name&cats=QualityControl).
+This provides at least one hundred different options to try! 👀 
+This is also an important step in not just single-cell, but also bulk RNA-seq as well, which has existed for over two decades.
+scRNA-seq has been around for over a decade, so there has been a lot of time to develop these methods across both technologies.
+
+[Cell Painting](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0080999) was first developed in 2013, meaning that image-based profiling and extraction of morphological profiles is a new field, but it still has been around for [over a decade](https://arxiv.org/abs/2405.02767).
+However, single-cell quality control for morphology profiling is not nearly as thoroughly developed. 😢
+
+This lack of development could be because of the lack of trust in this method, which is described in the 2017 paper, "[Data-analysis strategies for image-based cell profiling](https://www.nature.com/articles/nmeth.4397)".
+Caicedo et al. describes some methods of cell-level outlier detection, including model-free and model-based, but not specifically related to image-based profiling.
+Their main concern is that this method of quality control can assume homogeneity and remove interesting phenotypes (spoiler, we do run into this issue!).
+This is why most labs skip this step.
+Honestly, this concern applies to all quality control methods and underscores the importance of striving for continuous improvement.
+
+Currently, you are able to filter objects in CellProfiler using morphology measurements in its own module.
```

Comment:
what is an "object" - up to this point you're calling it "single-cell", maybe this works?

```suggestion
Currently, you are able to filter "objects" (some are single-cells, some might be debris) in CellProfiler using morphology measurements in its own module.
```

Reply:
Hmm objects is what the CellProfiler module is called since it can be nuclei, cells, or cytoplasm, just whatever being segmented. Let me correct this.